engine-tests/src/test/java/org/terasology/world/Zones/LayeredZoneRegionFunctionTest.java

```diff
@@ -38,11 +38,12 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.terasology.world.zones.LayeredZoneRegionFunction.LayeredZoneOrdering.ABOVE_GROUND;
```

Comment:
This block of static imports practically screams to be an exception to alpha sorting imports, so they could be listed in order based on how the layers would come one after another, but alas! I suspect trying to make an exception work would be more hassle than it would be worth seeing the "right" order :-)

Reply:
I agree that would be nice, but like you say it would probably be more hassle than its worth. I (and I'd imagine most people) just use Intellij's auto-imports, so it would require overriding that for specific classes, making sure everyone uses that override (including people who don't use Intellij), and keeping it up-to-date if the names/values change.

It would probably break at some point, creating annoying merge conflicts, and the potential gain is rather small (especially given that imports already have a logical and consistent ordering scheme that works well enough).